src/compiler/diagnosticMessages.json

```diff
@@ -1801,6 +1801,10 @@
         "category": "Error",
         "code": 1537
     },
+    "A `bigint` literal may not be used as a property name.": {

```

Comment:
Do we typically use backticks in diagnostic messages? I'm surprised to see them, personally. I tried searching and seems like we do, though inconsistency...

Reply:
I saw the same when looking through the messages :P